internal/configs/configurator.go

```diff
@@ -31,7 +31,7 @@ import (
 
 const (
 	pemFileNameForWildcardTLSSecret = "/etc/nginx/secrets/wildcard" // #nosec G101
-	appProtectBundleFolder          = "/etc/nginx/waf/bundles/"
+	appProtectBundleFolder          = "/etc/app_protect/bundles/"
```

Comment:
there are some tests in template_test.go and virtualserver_test.go which might need updating as well.

Reply:
I checked the tests dir and we don’t have any references to the bundles directory in there yet. We will need a test for this for v5 though.